terraform/modules/services/airflow/dags/india/runvnl-data-dag.py

```diff
@@ -22,9 +22,10 @@
 security_group = os.getenv("ECS_SECURITY_GROUP")
 cluster = f"india-ecs-cluster-{env}"
 
+region = 'india'
```

Comment:
Is it simplier, for the files that just had the region once, to be added directly to the f string?

Reply:
> Is it simplier, for the files that just had the region once, to be added directly to the f string?

Yes that could be done too, but i kept in mind that if in the future there are more DAGs added, it might help making it modular this way, what do you think?

Reply:
yea thats true, leave it how it is